### PR TITLE
Analisis mtie tdev

### PR DESCRIPTION
**Pull request by jnunyez**
This PRs:
- extends JSONEncoder for numpy array 
- dumps result of mtie and tdev analyzer for subsequent plotting in vse-sync-test
- adds unit test for changes done in MTIE and TDEV analyzer

### CODE REVIEW

File: tests/vse_sync_pp/analyzers/test_gnss.py [hunk 1]
```diff
@@ -526,6 +536,8 @@ class TestTimeDeviationAnalyzer(TestCase, metaclass=AnalyzerTestBuilder):
                     'stddev': 0,
                     'variance': 0,
                 },
+                'tdev_samples': np.array([0.]).tolist(),
```
Comment (nocturnalastro) [inline tests/vse_sync_pp/analyzers/test_gnss.py:539]: You don't need to wrap these in numpy arrays to then turn them back into a list.

File: tests/vse_sync_pp/analyzers/test_gnss.py [hunk 2]
```diff
@@ -445,6 +454,7 @@ class TestTimeDeviationAnalyzer(TestCase, metaclass=AnalyzerTestBuilder):
     constructor = TimeDeviationAnalyzer
     id_ = 'gnss/time-deviation'
     parser = 'gnss/time-error'
+    maxDiff = None
```
Comment (nocturnalastro) [inline tests/vse_sync_pp/analyzers/test_gnss.py:457]: What does this do?

File: tests/vse_sync_pp/analyzers/test_ppsdpll.py [hunk 3]
```diff
@@ -724,13 +741,15 @@ class TestTimeDeviationAnalyzer(TestCase, metaclass=AnalyzerTestBuilder):
             'analysis': {
                 'tdev': {
                     'units': 'ns',
-                    'min': 0,
-                    'max': 0,
-                    'range': 0,
-                    'mean': 0,
-                    'stddev': 0,
-                    'variance': 0,
+                    'min': 0.0,
```
Comment (nocturnalastro) [inline tests/vse_sync_pp/analyzers/test_ppsdpll.py:744]: Try getting rid of the .0 now we have the tdev_samples working ok.

File: tests/vse_sync_pp/analyzers/test_gnss.py (hunk 2 above)
Comment (jnunyez) [reply tests/vse_sync_pp/analyzers/test_gnss.py:457]: There now, nothing. Needs to go out :-)

**Review by nocturnalastro — APPROVED** [review]

**Pull request merged**